zeek/shaper.zed

```diff
@@ -1,57 +1,57 @@
 type port=uint16;
 type zenum=string;
-type id={orig_h:ip,orig_p:port,resp_h:ip,resp_p:port};
+type socket={orig_h:ip,orig_p:port,resp_h:ip,resp_p:port};
```

Comment:
The Zeek type for this is [`conn_id`](https://docs.zeek.org/en/lts/scripts/base/init-bare.zeek.html#type-conn_id). I think we should use that instead of our own name.

Reply:
(I don't think we made up the type `socket` but technically this is a `socketpair`.  Another term would be network `flow` though that usually include IP protocol.  That all said, I prefer the `conn_id` approach.)

Reply:
True, and I apologize for any implied shade!

"Our own name" was a poor choice of words. "A different name" is what I meant.

Reply:
Yes `conn_id` crossed my mind as well, so I've pushed a change to that.